civihr_employee_portal/civihr_leave_absences/civihr_leave_absences.module

```diff
@@ -81,6 +81,21 @@ function civihr_leave_absences_manager_leave_get_markup() {
 }
 
 /**
+ * Internal function for fetching custom markup from template/Angular Js to
+ * display number of pending leave requests.
+ *
+ * @return String
+ *    Custom markup from template file.
+ */
+function civihr_leave_absences_pending_leave_request_get_markup() {
```

Comment:
`civihr_leave_absences_pending_leave_request_get_markup`, `civihr_leave_absences_my_leave_get_markup`, `civihr_leave_absences_manager_leave_get_markup` are doing the same thing
```php
civicrm_resources_load('org.civicrm.reqangular', array('reqangular.min.js'));
civicrm_resources_load('uk.co.compucorp.civicrm.hrleaveandabsences', array('<module-name>.min.js'));

return civihr_leave_absences_load_file('<module-name-with-underscore>');
```

so I think you could refactor this and have just one function